logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/user/IsFileSharingEnabledUseCase.kt

```diff
@@ -11,27 +12,27 @@ import com.wire.kalium.logic.kaliumLogger
  */
 
 interface IsFileSharingEnabledUseCase {
-    operator fun invoke(): Boolean
+    operator fun invoke(): FileSharingEntity
 }
 
 class IsFileSharingEnabledUseCaseImpl(
     private val userConfigRepository: UserConfigRepository
 ) : IsFileSharingEnabledUseCase {
 
-    override operator fun invoke(): Boolean =
+    override operator fun invoke(): FileSharingEntity =
         userConfigRepository.isFileSharingEnabled().fold({
             when (it) {
                 StorageFailure.DataNotFound -> {
                     kaliumLogger.e("Data not found in IsFileSharingEnabledUseCase")
-                    false
+                    FileSharingEntity(null, null)
                 }
                 is StorageFailure.Generic -> {
                     kaliumLogger.e("Storage Error : ${it.rootCause} in IsFileSharingEnabledUseCase")
```

Comment:
```suggestion
                    kaliumLogger.e("Storage Error : ${it.rootCause} in IsFileSharingEnabledUseCase", it.rootCause)
```

So the whole StackTrace gets logged too.